src/plugins/workspace/server/integration_tests/duplicate.test.ts

```diff
@@ -0,0 +1,306 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// eslint-disable-next-line @osd/eslint/no-restricted-paths
+import * as exportMock from '../../../../core/server/saved_objects/export';
+import supertest from 'supertest';
+// eslint-disable-next-line @osd/eslint/no-restricted-paths
+import { SavedObjectsErrorHelpers } from '../../../../core/server/saved_objects';
+import { UnwrapPromise } from '@osd/utility-types';
+import { loggingSystemMock, savedObjectsClientMock } from '../../../../core/server/mocks';
+import { setupServer } from '../../../../core/server/test_utils';
+// eslint-disable-next-line @osd/eslint/no-restricted-paths
+import { workspaceClientMock } from '../../public/workspace_client.mock';
+import { registerDuplicateRoute } from '../routes/duplicate';
+import { createListStream } from '../../../../core/server/utils/streams';
+// eslint-disable-next-line @osd/eslint/no-restricted-paths
+import { mockUuidv4 } from '../../../../core/server/saved_objects/import/__mocks__';
+// eslint-disable-next-line @osd/eslint/no-restricted-paths
+import { createExportableType } from '../../../../core/server/saved_objects/routes/test_utils';
```

Comment:
not sure if we can export this at server level. Can you try and see if possible to remove `no-restricted-paths`